osu.Game/Overlays/OnlineOverlay.cs

```diff
@@ -41,8 +48,10 @@ protected OnlineOverlay(OverlayColourScheme colourScheme)
                         }
                     }
                 },
-                Loading = new LoadingLayer(true)
+                Loading = new LoadingLayer()
```

Comment:
intentional?

Reply:
originally was since i had this outside the OnlineView but i think it can be reverted now.